### ISSUE TITLE
Add option to adjust size of existing cursors?

### ISSUE THREAD

**Issue by haasn** [PandaTeemo/YoloMouse#13]
Instead of completely overriding `SetCursor`, could it be possible to hook the existing SetCursor call, take the resulting bitmap, upscale it, and then use that as the replacement cursor? That way we can basically use YoloMouse to make the game cursors larger without having to manually add them all.

I play at 4K resolution, and not all games have the ability to change the cursor size. So this would be really helpful.

**Comment by PandaTeemo** [comment]
Yes, this is definitely possible and on my todo list. You'd still have to
manually upscale each one using CTRL-ALT-+ though.
